### PR TITLE
v5.38.0 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/self-managed-changelog.md [hunk 1]
```diff
@@ -5,14 +5,156 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Added created-by property and improved performance with shared database connections. Focalboard 0.8.0 requires Mattermost v5.37+ due to a new database connection system.
+
+### Improvements
+
+#### User Interface (UI)
+ - Markdown formatting is now stripped from push notifications.
+ - Enabled the **Set Status** button if the custom status hasn't changed from currently set status.
+ - Improved default rendering of images inserted via the GIF picker.
+ - Small text changes were added to Direct and Group Message menus: 'Mute channel' and 'Edit Channel Header' now reads 'Mute Conversation' and 'Edit Conversation Header'.
+ - Changed the icon from a globe to a lock for private channels.
```
Comment (esethna) [inline source/install/self-managed-changelog.md:49]: I didn't think this is new?

**Review by esethna — APPROVED** [review]

File: source/install/self-managed-changelog.md (hunk 1 above)
Comment (amyblais) [reply source/install/self-managed-changelog.md:49]: @esethna It came from the release note that was added in this PR https://github.com/mattermost/mattermost-webapp/pull/8286.

**Review by wiersgallak — APPROVED** [review]

File: source/install/self-managed-changelog.md (hunk 1 above)
Comment (esethna) [reply source/install/self-managed-changelog.md:49]: @amyblais ah that PR is referring to the channel autocomplete, which will show private channels after the server PR is merged: https://github.com/mattermost/mattermost-server/pull/17804

No release note necessary until the feature goes in, and then the release note should say "Private channels are now available in the channel autocomplete list"

File: source/install/self-managed-changelog.md [hunk 2]
```diff
@@ -5,14 +5,155 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+
+3. MySQL versions below 5.7.7. Minimum support will now be for 5.7.12. This version introduced a native JSON data type that lets us improve performance and scalability of several database fields (most notably Users and Posts props). Additionally, version 5.6 (our current minimum version) reached [EOL in February 2021](https://www.mysql.com/support/eol-notice.html).
+
+4. Elasticsearch 5 and 6 - [versions 5.x reached EOL in March of 2019, and versions 6.x reached EOL in November 2020](https://www.elastic.co/support/eol). Our minimal supported version with Mattermost v6.0 will be Elasticsearch version 7.0.
+
+5. Windows 7 reached [EOL in January 2020](https://support.microsoft.com/en-us/windows/windows-7-support-ended-on-january-14-2020-b75d4580-2cc7-895a-2c9c-1466d9a53962). We will no longer provide support for the Desktop App issues on Windows 7.
+
+6. [DisableLegacyMFAEndpoint](https://docs.mattermost.com/administration/config-settings.html#disable-legacy-mfa-api-endpoint) configuration setting.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:122]:
```suggestion
6. [DisableLegacyMFAEndpoint](https://docs.mattermost.com/configure/configuration-settings.html#disable-legacy-mfa-api-endpoint) configuration setting.
```

File: source/install/self-managed-changelog.md [hunk 3]
```diff
@@ -5,14 +5,155 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+
+7. [Experimental Timezone](https://docs.mattermost.com/administration/config-settings.html#timezone) configuration setting.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:124]:
```suggestion
7. [Experimental Timezone](https://docs.mattermost.com/configure/configuration-settings.html#timezone) configuration setting.
```

File: source/install/self-managed-changelog.md [hunk 4]
```diff
@@ -5,14 +5,155 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+
+8. All legacy channel sidebar experimental configuration settings. We encourage customers using these settings to upgrade to v5.32 or later to access [custom, collapsible channel categories](https://mattermost.com/blog/custom-collapsible-channel-categories/) among many other channel organization features. The settings being deprecated include:
+
+   - [EnableLegacySidebar](https://docs.mattermost.com/administration/config-settings.html#enable-legacy-sidebar)
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:128]:
```suggestion
   - [EnableLegacySidebar](https://docs.mattermost.com/configure/configuration-settings.html#enable-legacy-sidebar)
```

File: source/install/self-managed-changelog.md [hunk 5]
```diff
@@ -5,14 +5,155 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+   - [ExperimentalTownSquareIsReadOnly](https://docs.mattermost.com/administration/config-settings.html#town-square-is-read-only-experimental)
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:129]:
```suggestion
   - [ExperimentalTownSquareIsReadOnly](https://docs.mattermost.com/configure/configuration-settings.html#town-square-is-read-only-experimental)
```

File: source/install/self-managed-changelog.md [hunk 6]
```diff
@@ -5,14 +5,155 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+   - [ExperimentalHideTownSquareinLHS](https://docs.mattermost.com/administration/config-settings.html#town-square-is-hidden-in-left-hand-sidebar-experimental)
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:130]:
```suggestion
   - [ExperimentalHideTownSquareinLHS](https://docs.mattermost.com/configure/configuration-settings.html#town-square-is-hidden-in-left-hand-sidebar-experimental)
```

File: source/install/self-managed-changelog.md [hunk 7]
```diff
@@ -5,14 +5,155 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+   - [EnableXToLeaveChannelsFromLHS](https://docs.mattermost.com/administration/config-settings.html#enable-x-to-leave-channels-from-left-hand-sidebar-experimental)
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:131]:
```suggestion
   - [EnableXToLeaveChannelsFromLHS](https://docs.mattermost.com/configure/configuration-settings.html#enable-x-to-leave-channels-from-left-hand-sidebar-experimental)
```

File: source/install/self-managed-changelog.md [hunk 8]
```diff
@@ -5,14 +5,155 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+   - [CloseUnusedDirectMessages](https://docs.mattermost.com/administration/config-settings.html#autoclose-direct-messages-in-sidebar-experimental)
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:132]:
```suggestion
   - [CloseUnusedDirectMessages](https://docs.mattermost.com/configure/configuration-settings.html#autoclose-direct-messages-in-sidebar-experimental)
```

File: source/install/self-managed-changelog.md [hunk 9]
```diff
@@ -5,14 +5,155 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+   - [ExperimentalChannelOrganization](https://docs.mattermost.com/administration/config-settings.html#sidebar-organization)
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:133]:
```suggestion
   - [ExperimentalChannelOrganization](https://docs.mattermost.com/configure/configuration-settings.html#sidebar-organization)
```

File: source/install/self-managed-changelog.md [hunk 10]
```diff
@@ -5,14 +5,155 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+   - [ExperimentalChannelSidebarOrganization](https://docs.mattermost.com/administration/config-settings.html#experimental-sidebar-features)
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:134]:
```suggestion
   - [ExperimentalChannelSidebarOrganization](https://docs.mattermost.com/configure/configuration-settings.html#experimental-sidebar-features)
```

File: source/install/self-managed-changelog.md [hunk 11]
```diff
@@ -5,14 +5,155 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+
+9. [All configuration settings previously marked as “Deprecated”](https://docs.mattermost.com/administration/config-settings.html#deprecated-configuration-settings).
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:136]:
```suggestion
9. [All configuration settings previously marked as “Deprecated”](https://docs.mattermost.com/configure/configuration-settings.html#deprecated-configuration-settings).
```

File: source/install/self-managed-changelog.md [hunk 12]
```diff
@@ -5,14 +5,155 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
...rShoukat96), [shred86](https://github.com/shred86), [skaramanlis](https://github.com/skaramanlis), [sowmiyamuthuraman](https://github.com/sowmiyamuthuraman), [sridhar02](https://github.com/sridhar02), [srkgupta](https://github.com/srkgupta), [stevemudie](https://github.com/stevemudie), [streamer45](https://github.com/streamer45), [stylianosrigas](https://github.com/stylianosrigas), [sudheerDev](https://github.com/sudheerDev), [svelle](https://github.com/svelle), [Szymongib](https://github.com/Szymongib), [TheoVitkovskiy](https://github.com/TheoVitkovskiy), [thePanz](https://github.com/thePanz), [TQuock](https://github.com/TQuock), [TRUNGTar](https://github.com/TRUNGTar), [tsabi](https://translate.mattermost.com/user/tsabi/), [utkuufuk](https://github.com/utkuufuk), [Vars-07](https://github.com/Vars-07), [venhaus](https://github.com/venhaus), [vijaynag-bs](https://github.com/vijaynag-bs), [webchick](https://github.com/webchick), [wget](https://github.com/wget), [wiersgallak](https://github.com/wiersgallak), [wiggin77](https://github.com/wiggin77), [Willyfrog](https://github.com/Willyfrog), [Yohannesseifu](https://github.com/Yohannesseifu), [YushiOMOTE](https://github.com/YushiOMOTE)
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:155]: cwarnermm is missing from this list :)

**Review by cwarnermm — COMMENTED** [review]
Great updates! I've provided URL updates and review feedback inline.

**Pull request merged**